src/pages/[platform]/build-a-backend/storage/list-files/index.mdx

```diff
@@ -109,6 +109,43 @@ function processStorageList(response) {
 
 This places each item's data inside a special `__data` key.
 
+### Exclude Subpaths
+
+When objects are organized using a prefix you can return only those objects within the `path` you've provided, and not those within nested subpaths.
+
+For example, given the following keys in your `path` you may want to return only the jpg objects, and not the "vacation" subpath:
+
+
+** `photos/photo1.jpg`
+** `photos/photo2.jpg`
+** `photos/vacation/`
+
+
+This can be accomplished with the `subpathStrategy` option:
+
+```js
+const result = await list({ 
+  path: "photos/",
+  options:{
+    subpathStrategy: { strategy:'exclude' }
+  }
+});
+```
```

Comment:
can we also include an example of the output. since this result does contain the `excludedSubPaths` as well?

Reply:
It's possible to extract the object. e.g. `const =  { excludedSubpaths } = await list(..`

Reply:
yes that would work.